be/src/column/object_column.cpp

```diff
@@ -124,6 +124,18 @@ void ObjectColumn<T>::append_default(size_t count) {
     }
 }
 
+template <typename T>
+Status ObjectColumn<T>::update_rows(const Column& src, const uint32_t* indexes) {
```

Comment:
I found HyperLogLog and BitmapValue support assign operator. But PercentileValue does not.
You should check it will right or not.

Reply:
> I found HyperLogLog and BitmapValue support assign operator. But PercentileValue does not. You should check it will right or not.

done，add ut for percentile_column in https://github.com/StarRocks/starrocks/pull/2870/commits/9f6799c36bf7c14c4ae6f1765ba5e50b231ad9b6